base/c.jl

```diff
@@ -110,8 +106,17 @@ end
 # symbols are guaranteed not to contain embedded NUL
 convert(::Type{Cstring}, s::Symbol) = Cstring(unsafe_convert(Ptr{Cchar}, s))
 
-# FIXME: this should be handled by implicit conversion to Cwstring, but good luck with that
 if is_windows()
+"""
+    Base.cwstring(s)
+
+Converts a string `s` to a NUL-terminated `Vector{Cwchar_t}`, suitable for passing to C
+functions expecting a `Ptr{Cwchar_t}`. The main advantage of using this over the implicit
+conversion provided by `Cwstring` is if the function is called multiple times with the
+same argument.
+
+This is only available on Windows.
```

Comment:
Is it? From what I can tell, the following are equivalent:
```julia
ccall(....,Cwstring, str)
```
or
```julia
cwstr = cwstring(str)
ccall(....,Ptr{Cwchar_t}, str)
```
the advantage of the 2nd one is that if you have to call the function multiple times, you only have to do the conversion once.